project3/styles1.css

```diff
@@ -55,14 +57,20 @@ main {
     color: #eaecf0;
     padding-left: 16px;
     padding-right: 16px;
+    padding-bottom: 50px;
     text-align: left;
 }
 
+input[type="text"]:not(:placeholder-shown):invalid,
+input[type="email"]:invalid,
+input[type="number"]:invalid {
+    border-bottom: #71f79f;
+}
+
 legend {
     font-family: Teko;
     font-weight: 500;
     font-size: 40px;
-
 }
 
 fieldset {
```

Comment:
On line 83 below, the `font-weight` should not have a `px` value. Same on line 91.